package.json

```
@@ -48,7 +48,7 @@
   "dependencies": {
     "aws-sdk": "2.41.0",
     "babel-runtime": "6.23.0",
-    "lodash": "4.17.11",
+    "lodash": "^4.17",
```

@rmbl thanks! I believe that ^ will allow for both patch and minor upgrades (see https://www.npmjs.com/package/semver). Generally, I'm of the philosophy that the more pinning the better, but since lodash appears to be pretty active at the patch level, I'm OK with a ~4.17.0 specifier
```suggestion
    "lodash": "~4.17.0",
```